resources/views/mailformat/send.blade.php

```diff
@@ -136,8 +125,98 @@ class="form-control"
     </ul>
   </fieldset>
 
+  <fieldset>
```

Comment:
Ff class="mb-3" toevoegen, anders staat dat versturen knopje zo tegenaan geplakt.

Reply:
```suggestion
  <fieldset class="mb-3">
```
Kan ook zo jongeman 
![FriendsJoeyTribbianiGIF](https://user-images.githubusercontent.com/23651869/121338222-97073a80-c91d-11eb-8ac1-d478b4747387.gif)